llama-index-integrations/embeddings/llama-index-embeddings-huggingface/llama_index/embeddings/huggingface/base.py

```diff
@@ -30,6 +31,10 @@
 logger = logging.getLogger(__name__)
 
 
+@deprecated(
+    "Deprecated in favor of `TextGenerationInferenceEmbedding` from `llama-index-embeddings-text-generation-inference` which should be used instead.",
+    action="always",
+)
```

Comment:
I think this deprecation notice was meant to be placed on TGI rather than the basic hf embeddings